src/find_missing_words/gui/config/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""
+Config dialog entry point.
+This module will load the config initially for the tabs.
+Each tab of the config will write to the config itself.
+Write actions will be triggered here when the user clicks the dialog "Save All" button.
+"""
+from aqt import mw
+from aqt.qt import *
+
+from ..forms import config as config_form
+from .search_tab import SearchTab
+from .note_creation_tab import NoteCreationTab
+
+
+class ConfigDialog(QDialog):
```

Comment:
I thought there was some Python convention to do minimal work like loading 1 thing and kicking that off? :thinking: Have I merely imagined this? :sweat_smile:

Reply:
Not sure I see what you mean? Lol 
Are you talking about this class?